js/app/modules/arrangement/sideBySide.js

```diff
@@ -51,66 +72,73 @@ const _SideBySideLayout = new Knowledge.Class({
         if (all_cards.length === 0)
             return [0, 0];
 
-        let [min, nat] = all_cards[0].get_preferred_width();
+        let [, first_nat] = all_cards[0].get_preferred_width();
+        let [, button_nat] = this._popover_button.get_preferred_width();
+
+        let min = first_nat + SPACING_MIN + button_nat;
+        let nat = 0;
 
-        nat += all_cards.slice(1).reduce((accum, card) => {
+        all_cards.forEach ((card) => {
             let [, card_nat] = card.get_preferred_width();
-            return accum + card_nat;
-        }, 0);
+            nat += card_nat + SPACING_MAX;
+        });
 
-        nat += _HorizontalSpacing.XLARGE * (all_cards.length - 1);
-        return [min, nat];
+        return [min, nat - SPACING_MAX];
+    },
+
+    _ensure_card_parent: function (card, parent) {
+        let current_parent = card.get_parent();
+
+        if (current_parent != parent) {
+            current_parent.remove(card);
```

Comment:
reparent() is deprecated since 3.14, BTW I did not added a reference since GJS holds one anyways